content/syllabuses/employed-java-skills-test.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Employed Java skills test
+---
+
+- {{% contentlink path="projects/katas/level-1" flavour="java" %}}
```

Comment:
This looks way too hard. Should just be katas 0.

I thin you swapped the files around. 

Post skills test technical content : We need folks to nail git next, then continue with code